integration-cli/check_test.go

```diff
@@ -125,17 +74,17 @@ func TestMain(m *testing.M) {
 	for _, img := range images {
 		protectedImages[img] = struct{}{}
 	}
-	if !isLocalDaemon {
-		fmt.Println("INFO: Testing against a remote daemon")
-	} else {
+	if testEnv.LocalDaemon() {
 		fmt.Println("INFO: Testing against a local daemon")
+	} else {
+		fmt.Println("INFO: Testing against a remote daemon")
 	}
 	exitCode := m.Run()
 	os.Exit(exitCode)
 }
 
 func Test(t *testing.T) {
-	if daemonPlatform == "linux" {
+	if testEnv.DaemonPlatform() == "linux" {
```

Comment:
Yes, I didn't changed those *yet*, it's just the refactor here, but that would be an enhancement 👼